course05/homework01/skeleton_ivo/string.h

```diff
@@ -0,0 +1,30 @@
+#include <string>
+#include <ostream>
+#include <istream>
```

Comment:
You only need the declarations in the header file; so you should include iosfwd, and these ones in the cc file